### PR TITLE
update text recognition 2

### PR DESCRIPTION
**Pull request by morkovka1337**
1. Add link to the omz page for 0016 model
2. Replace os.system with urllib.request

### CODE REVIEW

**Comment by morkovka1337** [comment]
run tests

**Comment by morkovka1337** [comment]
run tests

**Comment by morkovka1337** [comment]
run tests

**Comment by morkovka1337** [comment]
run tests

**Comment by morkovka1337** [comment]
run tests

**Comment by morkovka1337** [comment]
run tests

**Comment by morkovka1337** [comment]
run tests

**Comment by morkovka1337** [comment]
run tests

**Comment by morkovka1337** [comment]
run tests

**Comment by morkovka1337** [comment]
@Ilya-Krylov, maybe it will be more correct to reduce batch size and number of workers in the separate PR, and when it is merged, rebase this, what do you think?

File: misc/pytorch_toolkit/chest_xray_screening/tests/test_train.py [hunk 1]
```diff
@@ -35,21 +35,15 @@ def setUpClass(cls):
                 image_path, transform=True)
             cls.data_loader_train = DataLoader(
                 dataset=dataset_train,
-                batch_size=2,
```
Comment (Ilya-Krylov) [inline misc/pytorch_toolkit/chest_xray_screening/tests/test_train.py:38]: why do you update it?

Comment (morkovka1337) [reply misc/pytorch_toolkit/chest_xray_screening/tests/test_train.py:38]: otherwise train tests for this model fail on the CI

Comment (Ilya-Krylov) [reply misc/pytorch_toolkit/chest_xray_screening/tests/test_train.py:38]: what is the value now?

**Pull request merged**